recipes/vs2008_runtime/bld.bat

```diff
@@ -0,0 +1,25 @@
+if "%ARCH%" == "32" (set "ARCH_DIR=x86")
+if "%ARCH%" == "64" (set "ARCH_DIR=amd64")
+
+dir /a:d /o:-n /s /b C:\Windows\WinSxS\%ARCH_DIR%_microsoft.vc90.openmp* > vcomp90_locs.txt
+type vcomp90_locs.txt
+
+set /p VCOMP_DIR=<vcomp90_locs.txt
+echo "%VCOMP_DIR%"
+
+for %%F in ("." "bin") do (
+    cmake -G "%CMAKE_GENERATOR%" ^
+          -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+          -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS:PATH="%VCOMP_DIR%\vcomp90.dll" ^
```

Comment:
Manually add in the `vcomp90` library based on what we found above. This ensures CMake will install it.